OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -102,14 +102,16 @@ public class ConnectionFailedLogic : ChromeLogic
 		bool passwordOffsetAdjusted;
 
 		[ObjectCreator.UseCtor]
-		public ConnectionFailedLogic(Widget widget, ModData modData, OrderManager orderManager, NetworkConnection connection, string password, Action onAbort, Action<string> onRetry)
+		public ConnectionFailedLogic(Widget widget, ModData modData, OrderManager orderManager, NetworkConnection connection, string password, Action onAbort, string abortTextOverwrite, Action<string> onRetry)
```

Comment:
Maybe create an `Action onQuit` and if it is not null show a `QUIT_BUTTON` button?

Reply:
definitely an option